web/components/FcPaneMap.vue

```diff
@@ -225,26 +270,16 @@ export default {
       'drawerOpen',
       'legendOptions',
       'locationMode',
+      'locations',
+      'locationsEdit',
```

Comment:
we don't use this anywhere right now!  in `locationsGeoJson`, we should be toggling between `this.locations` and `this.locationsEdit` depending on whether `this.locationMode === LocationMode.MULTI_EDIT`.